### DISCUSSION TITLE
How to test _error/500 page locally

### DISCUSSION THREAD

**Discussion by ffpetrovic-ct** [vercel/next.js#35575] [Help]
How can the custom `_error.tsx` page be tested locally, if the overlay shows up instead of the page? This overlay can't be `Escape`'d or clicked away from either. And `npm run build && npm run start` is not an acceptable solution, since a watchable workflow is already available, `npm run dev`.

**Comment by icyJoseph** [comment]
I think this is a rather difficult (impossible?) thing to do: https://github.com/vercel/next.js/discussions/13387

From the [docs](https://nextjs.org/docs/advanced-features/custom-error-page#reusing-the-built-in-error-page)

> `pages/_error.js` is only used in production. In development you’ll get an error with the call stack to know where the error originated from.

Of course, 404's can be used to test the `_error.tsx` page by just going to a non existing route, but I image you want to test actual page failures. 

---

I think the idea behind this is that, if the error comes from rendering, a Boundary should catch it, and if it comes from `gSSP`, then you should formally handle it on the server.

I understand that, maybe people have logic and what not on the error pages, styling and such, that one wants to see and test, and this makes it rather difficult.
